docworks-dts/lib/multiple-files/serviceUtils.js

```diff
@@ -0,0 +1,109 @@
+const has_ = require('lodash/has')
+const set_ = require('lodash/set')
+const isEmpty_ = require('lodash/isEmpty')
+const { SUB_SERVICES_KEY } = require('./constants')
+
+const isRootService = service => !service.memberOf
+const withServicesProperty = service =>
+	Object.assign({ [SUB_SERVICES_KEY]: {} }, service)
+
+const isEmptyInterface = service =>
+	isEmpty_(service.properties) && isEmpty_(service.operations)
+
+const isEmptyNamespace = service =>
+	isEmpty_(service.services) &&
+	isEmpty_(service.messages) &&
+	isEmpty_(service.callbacks)
+
+const getServiceSummary = service =>
+	service && service.docs && service.docs.summary ? service.docs.summary : ''
```

Comment:
This can also be a part of the documentationGenerator, but your call since it's service specific.